x-pack/test/functional/apps/machine_learning/data_frame_analytics/classification_creation.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  describe('classification creation', function() {
+    this.tags(['smoke']);
+    before(async () => {
+      await esArchiver.load('ml/bm_classification');
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+      await esArchiver.unload('ml/bm_classification');
+    });
+
+    const testDataList = [
+      {
+        suiteTitle: 'bank marketing',
+        jobType: 'classification',
+        jobId: `egs_1_${Date.now()}`,
+        jobDescription: 'This is the classification job description',
```

Comment:
Suggestion:
Comparing the df analytics job descriptions with the anomaly detection ones, lets me think we might want to describe our jobs a bit better here. Maybe something like
```
Classification job based on 'bank-marketing' dataset with dependentVariable 'y' and trainingPercent '20'
```
(similar for regression and outlier detection jobs)

Reply:
Updated in 5c3fa853e3c33ad4e90c8ed44b4ea0885ca20652